apps/ideas/forms.py

```diff
@@ -177,6 +182,25 @@ class Meta:
             'members'
         ]
 
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+        super().__init__(*args, **kwargs)
+        help_pages = HelpPages.for_site(self.request.site)
```

Comment:
There are other places (like the [email](https://github.com/liqd/adhocracy4/blob/master/adhocracy4/emails/base.py#L19)), where we assume that there is only one site (with id), to prevent passing the request further down. I guess we could do here the same.